### PR TITLE
fixed "learning rate is illegal" warning

### PR DESCRIPTION
**Pull request by leacoleaco**
when I training the data with tensorflow 1.3， the console print the warning text ”learning rate is illegal“ .  so i find and changed this word

### CODE REVIEW

**Comment by leacoleaco** [comment]
@googlebot I signed it!
